libminifi/test/resources/TestHTTPGetSecure.yml

```diff
@@ -77,11 +77,11 @@ Controller Services:
       class: SSLContextService
       Properties:
         Client Certificate:
-            - value: cn.crt.pem
+            - value: cn.p12
```

Comment:
This fixes the test failure that happened on some mac versions, so the test could be reenabled.